lib/utils/html-fn.html

```diff
@@ -0,0 +1,61 @@
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="boot.html">
+<script>
+  (function() {
+    'use strict';
+
+    /**
+     * @param {*} value Object to stringify into HTML
+     * @return {string} HTML stringified form of `obj`
+     */
+    function htmlValue(value) {
+      if (value instanceof HTMLTemplateElement) {
+        return /** @type {!HTMLTemplateElement} */(value).innerHTML;
+      } else {
+        return String(value);
+      }
+    }
+
+    /**
+     * Small tagged template string function to facilitate building a `<template>` element.
```

Comment:
I don't think the public documentation needs "small". I'd go with something like:

A template literal tag[1] that creates an HTML `<template>` element from the contents of the string. This allows you to write a Polymer Template in JavaScript.

Interpolated values are converted to strings when the template is created, they are not intended as a replacement for Polymer data-binding. There is special support for HTMLTemplateElement values, allowing for easy composition of superclass or partial templates.

[1]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_template_literals

Reply:
I'll just copy that in there, thank you.